### ISSUE TITLE
Facing problem while opening word application such As Microsoft Word or BarahaPad

### ISSUE THREAD

**Issue by narayan-b-raikar** [nvaccess/nvda#14545]
-->

#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu? Yes.

As suggested previous details, recently, After I  installed the new version of NVDA version 2022.4, applications such as microsoft word, BarahaPad, Whatsapp, WordPad etc. are not coming into the focus of NVDA Screen-Reader after opening. So I have to restart NVDA every time. Even though the application comes into focus after restarting, if I try to switch to another window through commands such as Alt+Tab, Windows+Tab, Windows+M etc., NVDA Screen-Reader remains inactive without speaking. I cannot get extra log information while this problem remains active. Will try to upload Log information If I am Able to get this.
Thank you

**Comment by narayan-b-raikar** [comment]
Sorry For my previous Mistakes. Actually I am facing the mentioned problem while All addons are enabled. I am unable to get Log Info While Addons are enable. Therefore, I am unable to find that which addon is causing this problem. So, Still I collected some log info which are generated while All addons are disabled. 
[NVDA Log Info of BarahaPad.txt](https://github.com/nvaccess/nvda/files/10421611/NVDA.Log.Info.of.BarahaPad.txt)
[NVDA Log Info with Microsoft Word.txt](https://github.com/nvaccess/nvda/files/10421612/NVDA.Log.Info.with.Microsoft.Word.txt)

**Comment by narayan-b-raikar** [comment]
[NVDA Log Info With BarahaPad while All Addons are enabled.txt](https://github.com/nvaccess/nvda/files/10421642/NVDA.Log.Info.With.BarahaPad.while.All.Addons.are.enabled.txt)
[NVDA Log Info With BarahaPad while All Addons are enabled2.txt](https://github.com/nvaccess/nvda/files/10421643/NVDA.Log.Info.With.BarahaPad.while.All.Addons.are.enabled2.txt)

**Comment by seanbudd** [comment]
@narayan-b-raikar , if this issue is with an add-on, it is not within the scope of the NVDA GitHub and we cannot provide an investigation.
